src/idd/idd.markers.js

```diff
@@ -26,8 +26,8 @@ InteractiveDataDisplay.Markers = function (div, master) {
 
     var _data = {};
     var _shape;
-    var _colorPalette, _sizePalette;
-    var _colorRange, _sizeRange;
+    var _colorPalette, _uncertainColorPalette, _sizePalette;
```

Comment:
@ElenaPochernina why did you need another color and size palettes in addition to the existing? 
Is there a case when they are used together in a same plot?